src/routes/components/radio/+page.svelte

```diff
@@ -0,0 +1,130 @@
+<script lang="ts">
+	import radio from '$lib/components/ui/radio-button/radio.svelte?raw'
+	import radioGroup from '$lib/components/ui/radio-button/radio-group.svelte?raw'
+	import Code from '$lib/components/custom/code.svelte'
+	import TabBar from '$lib/components/custom/tab-bar/tab-bar.svelte'
+	import TabContent from '$lib/components/custom/tab-bar/tab-content.svelte'
+	import { onMount } from 'svelte'
+	import Select from '$lib/components/ui/select/select.svelte'
+	import Option from '$lib/components/ui/select/option.svelte'
+	import ComponentTemplate from '$lib/components/custom/component-template.svelte'
+	import Radio from '$lib/components/ui/radio-button/radio.svelte'
+	import RadioGroup from '$lib/components/ui/radio-button/radio-group.svelte'
+
+	type Dimension = 'default' | 'large' | 'compact' | 'small'
+	type Layout = 'vertical' | 'horizontal'
+
+	let radioCss: string = $state('Loading...')
+	let radioGroupCss: string = $state('Loading...')
+
+	let dimension: Dimension = $state('default')
+	let layout: Layout = $state('vertical')
+
+	// Svelte compiler breaks when it finds closing script tag, hence the need to make the template literal to have two parts
+	let useCode = $derived(
+		`<script lang="ts">
+    import RadioGroup from '$lib/components/ui/radio-button/radio-group.svelte'
+    import Radio from '$lib/components/ui/radio-button/radio.svelte'
+</script` +
+			`>
+
+<RadioGroup dimesion="${dimension}" layout="${layout}" name="rb">
+    {#snippet helperText()}
+        This is an optional helper text
+    {/snippet}
+    {#snippet label()}
+        Radio group label
+    {/snippet}
+    <Radio>Radio label</Radio>
+    <Radio>Radio label</Radio>
+</RadioGroup>
+`,
+	)
+
+	onMount(async () => {
+		const radio = await fetch('/generated/css/ui/radio-button/radio.css')
+		const radioGroup = await fetch('/generated/css/ui/radio-button/radio-group.css')
+		radioCss = await radio.text()
+		radioGroupCss = await radioGroup.text()
+	})
+</script>
+
+{#snippet description()}
+	Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quasi vitae explicabo, distinctio velit
```

Comment:
As an example what could be useful here please check out the checkbox component page. At least the sizes can be enumerated, or show the difference between vertical and horizontal layout.